src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+/**
+ * A type of tasks, have a specific date(deadline)
+ */
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Perhaps this variable name could be slightly more descriptive? Something like "doByDate"? 😅